contracts/prize-strategy/PrizeStrategy.sol

```diff
@@ -852,12 +866,15 @@ contract PrizeStrategy is PrizeStrategyStorage,
   }
 
   modifier requireCanStartAward() {
+    require(_isPrizePeriodStarted(), "PrizeStrategy/prize-period-not-started");
     require(_isPrizePeriodOver(), "PrizeStrategy/prize-period-not-over");
     require(!isRngRequested(), "PrizeStrategy/rng-already-requested");
     _;
   }
 
   modifier requireCanCompleteAward() {
+    require(_isPrizePeriodStarted(), "PrizeStrategy/prize-period-not-started");
```

Comment:
✂️ 

We already have isPrizePeriodOver

Reply:
done